be/test/formats/parquet/encoding_test.cpp

```diff
@@ -28,6 +32,7 @@ class ParquetEncodingTest : public testing::Test {
     ~ParquetEncodingTest() override = default;
 };
 
+// clang-format off
```

Comment:
remove these lines to allow clang-format check taking effect.

Reply:
got it.